src/style.css

```diff
@@ -1,3 +1,53 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+
+  --my-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.imgContainer {
+  margin: 10px 0 10px 50px;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both)